include/nbl/core/math/glslFunctions.tcc

```diff
@@ -10,6 +10,10 @@
 #include "nbl/core/math/floatutil.tcc"
 #include "matrix4SIMD.h"
 
+#if defined(__clang__)
+#include <boost/math/special_functions/bessel.hpp>
+#endif //__clang__
+
```

Comment:
you might be breaking stuff for @pollend , best you check for AppleClang instead